src/test/java/com/google/cloud/tools/appengine/cloudsdk/PathResolverTest.java

```diff
@@ -31,6 +46,51 @@ public void testResolve() {
   @Test
   public void testGetRank() {
     Assert.assertTrue(resolver.getRank() > 0);
-  }  
-  
+  }
+
+  @Test
+  public void testGetLocationFromLink_valid() throws IOException {
+    Path sdkHome = temporaryFolder.newFolder().toPath();
+    Path bin = Files.createDirectory(sdkHome.resolve("bin"));
+    Path gcloud =  Files.createFile(bin.resolve("gcloud"));
+    Files.createSymbolicLink(temporaryFolder.getRoot().toPath().resolve("gcloud"), gcloud);
```

Comment:
So these tests seem to work if you run under Administrator in windows server 2016. I've updated the test assumptions on these symlink test classes.

From windows testing, PathResolver will skip 3 tests if windows(non-admin), run all tests in admin mode.